plugins/become/sudosu.py

```diff
@@ -55,6 +55,18 @@
             ini:
               - section: sudo_become_plugin
                 key: password
+        alt_method:
+            description: 
+              - Whether to use an alternative method to call sudosu. 
+              - Use this when the default one is not working on your system.
+            required: false 
```

Comment:
I assume this is a boolean?
```suggestion
            required: false 
            type: boolean
```